packages/flutter/test/widgets/container_test.dart

```diff
@@ -746,9 +773,7 @@ void main() {
     );
 
     await tester.pumpWidget(
-      RepaintBoundary(
-        child: Padding(padding: const EdgeInsets.all(30.0), child: container),
-      ),
+      RepaintBoundary(child: Padding(padding: const EdgeInsets.all(30.0), child: container)),
```

Comment:
(Huh, it's interesting that this unrelated formatting change appears in the diff. Do you know why that happened? This is the kind of thing that an auto-formatter is supposed to promise to prevent…)

Reply:
It might have been due to an incorrect version of Flutter SDK. Seems like I fixed it